src/ngraph/op/util/unary_elementwise_arithmetic.cpp

```diff
@@ -18,6 +18,16 @@
 
 using namespace ngraph;
 
+op::util::UnaryElementwiseArithmetic::UnaryElementwiseArithmetic()
```

Comment:
I think this is extraneous to this PR, right?

Reply:
Gone.